openassessment/management/commands/collect_ora2_data.py

```diff
@@ -0,0 +1,67 @@
+"""
+Command to retrieve all ORA2 data for a course in a .csv.
+
+
+This command differs from upload_oa_data in that it places all the data into one file.
+
+Generates the same format as the instructor dashboard downloads.
+"""
+import csv
+from optparse import make_option
+import os
+
+from openassessment.data import AggregateOraData
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Query aggregated open assessment data, write to .csv
+    """
+
+    help = ("Usage: collect_ora2_data <course_id> --output-dir=<output_dir>")
+    args = "<course_id>"
+
+    option_list = BaseCommand.option_list + (
+        make_option('-o', '--output-dir',
+                    action='store', dest='output_dir', default=None,
+                    help="Write output to a directory rather than stdout"),
+    )
+
+    def handle(self, *args, **options):
+        """
+        Run the command.
+        """
+        if not args:
+            raise CommandError("Course ID must be specified to fetch data")
+
+        course_id = args[0]
```

Comment:
The problem is that ORA2 doesn't really have a concept of what a valid or invalid course id is. It's just a string in the code right now. I wasn't sure I wanted to add opaque-keys as a dependency just for this one minor use case.